pkg/cvo/cvo.go

```diff
@@ -31,6 +32,12 @@ const (
 //   - checking for updates and writing the available updates into the CVO's
 //     OperatorStatus.
 func StartWorkers(stopCh <-chan struct{}, config *rest.Config) {
+	kubeClient, err := v1.NewForConfig(config)
+	if err != nil {
+		glog.Errorf("Failed to create Kubernetes client: %v", err)
```

Comment:
nit: This is an `apps/v1` client. so maybe `Failed to create Kubernetes client (apps/v1)` like ones below